app/app.go

```diff
@@ -942,7 +942,7 @@ func New(
 
 	// RegisterUpgradeHandlers is used for registering any on-chain upgrades.
 	// Make sure it's called after `app.ModuleManager` and `app.configurator` are set.
-	app.RegisterUpgradeHandlers()
+	// app.RegisterUpgradeHandlers()
```

Comment:
app.RegisterUpgradeHandlers() seems to be doing something similar to app.setupUpgradeHandlers(), let's de-duplicate and use a single function.